torch/onnx/_internal/fx/registration.py

```diff
@@ -14,9 +13,6 @@
 # 'import onnx' inside of dynamo_export (by way of _assert_dependencies).
 if TYPE_CHECKING:
     import onnxscript  # type: ignore[import]
-    from onnxscript.function_libs.torch_lib import registration  # type: ignore[import]
-
-OpsetVersion = int
 
 
```

Comment:
I would suggest renaming `SymbolicFunction` to something else. The `Symbolic` term is heavily overloaded lately, especially with dynamic shape (which has symbolic or static shape) and fake tensor (which has the idea of symbolic tracing)

Maybe something more neutral like `class ONNXFunction` or `OpImpl`, `ONNXScriptFunction` would be more meaningful for the user that will implement custom ops in the future

Reply:
Good point

Reply:
Go with `ONNXFunction`